SU-SOE/stanford_bean_types
# SOE-3382: hide file insert

# READY FOR REVIEW

# Summary
_See ticket:_ https://stanfordits.atlassian.net/browse/SOE-3382
This gives the editor the choice of how the file will be displayed. See Before and After screenshots below.

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff


# Steps to Test
- Checkout this branch: `git checkout 7.x-2.x-dev-SOE-3382-hide-file-insert`
- Revert feature: `drush fr stanford_bean_types -y --force`
- Clear registry and caches: `drush rr ; drush cc all`
- Navigate to `catalog-patterns/blocks-and-beans/postcard-blocks`
- Edit postcard **Sample Postcard Block 1**
- Verify you see  the **display** checkbox:
![screen shot 2018-08-29 at 11 33 58](https://user-images.githubusercontent.com/284440/44807948-baeb9e00-ab7f-11e8-96f8-93f96218e5be.png)
- Deselect the **display** checkbox and save
- Verify the file no longer displays twice:
Before:
![screen shot 2018-08-29 at 11 37 27](https://user-images.githubusercontent.com/284440/44808085-27669d00-ab80-11e8-9bce-f8e5fc14f240.png)
After:
![screen shot 2018-08-29 at 11 37 56](https://user-images.githubusercontent.com/284440/44808096-2e8dab00-ab80-11e8-908a-8cb2f569f2e7.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3382
## Related PRs

## More Information

## Folks to notify
`@minorwm `
`@annawatt `
`@kerri-augenstein `
`@boznik `
`@cjwest`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
